index.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="css/main.css">
+  <link href='http://fonts.cdnfonts.com/css/cocogoose' rel="stylesheet" />
+  <link href='http://fonts.cdnfonts.com/css/lato' rel="stylesheet" />
+  <title>Art Blog ~ Home Page</title>
+</head>
+
+<body>
+  <div class="container">
+
+    <!-- Top Navigation -->
+    <section class="navigation">
+      <nav class="navigation__top-nav">
+        <ul>
+          <li class="nav-item">
+            <svg class="nav-item__icon icon-menu">
+              <use xlink:href="assets/Icons/sprite.svg#icon-facebook"></use>
+            </svg>
+          </li>
+          <li class="nav-item">
+            <svg class="nav-item__icon icon-cross">
+              <use xlink:href="assets/Icons/sprite.svg#icon-twitter"></use>
+            </svg>
+          </li>
+          <li class="nav-item">
+            <a href="" class="nav-link">English</a>
+          </li>
+          <li class="nav-item">
+            <a href="" class="nav-link">My Page</a>
+          </li>
+          <li class="nav-item">
+            <a href="" class="nav-link">Logout</a>
+          </li>
+        </ul>
+      </nav>
+    </section>
+
+    <!-- Header Section -->
+    <header class="header">
+
+      <!-- Headre Nav -->
+      <section class="header-nav">
+        <div class="header-nav-container">
+          <div class="logo-box" id="logo-box-nav"><span>Art</span>~Blog</div>
+          <nav class="header-nav-container__nav-bar">
+            <div class="header-nav-container__nav-bar--icon" id="hamberger">
+              <svg class="header-nav-container__nav-bar--icon" id="icon-menu">
+                <use xlink:href="assets/Icons/sprite.svg#icon-menu"></use>
+              </svg>
+              <svg class="header-nav-container__nav-bar--icon" id="icon-cross">
+                <use xlink:href="assets/Icons/sprite.svg#icon-cross"></use>
+              </svg>
+            </div>
+            <ul class="header-nav-container__nav-bar--link">
+              <li class="nav-link"><a href="pages/about.html">About</a></li>
+              <li class="nav-link"><a href="">Program</a></li>
+              <li class="nav-link"><a href="">Join</a></li>
+              <li class="nav-link"><a href="">Sponsor</a></li>
+              <li class="nav-link"><a href="">News</a></li>
```

Comment:
- [ ] **Optional:** You have done an amazing job. Kudos 👏👏👏
However, I have noticed that your website doesn't have a navigation link to the homepage. If a user navigates to the `About` page, it may be difficult for them to get back to the homepage.  _**Kindly include it!**_